PalindromeLab/README.md

```diff
@@ -0,0 +1,24 @@
+<h1>Python Autograder Template with Palindrome Lab</h1>
```

Comment:
I would prefer if you simply titled it "Palindrome Lab"

Reply:
Renamed it.